### ISSUE TITLE
retrieve record to exclude the link/document field.

### ISSUE THREAD

**Issue by martingg88** [orientechnologies/orientjs#142]
How to exclude the link/document field for following query as it had automatically retrieve the link record from  other class. 

select from thing

 { '@type': 'd',
   '@class': 'thing'
   lowest_rate: 68,
   translate: object (note: it try to retrieve document from other class. how to avoid it?)
}

**Comment by maggiolo00** [comment]
@martingg88 

can you explain it better with an example?

Do you want to exclude the translate object?

**Comment by martingg88** [comment]
yes. how to exclude translate object or any object linked by edge.

**Comment by maggiolo00** [comment]
hi @martingg88 
have you look here?
http://orientdb.com/docs/2.1/Fetching-Strategies.html

**Comment by martingg88** [comment]
can we have a simple documentation for fetchplan as we not quite understand how it work after reading the existing documentation. That will be great if you can give us some simple example. 
Many thanks to you.
